addons/html_builder/static/src/builder/options/alignment_option.xml

```diff
@@ -11,4 +11,16 @@
     </BuilderRow>
 </t>
 
+<t t-name="html_builder.VerticalAlignmentOption">
+    <!-- TODO add title tooltip row Vertical Alignment-->
+    <BuilderRow label.translate="Vert. Alignment" level='level'>
```

Comment:
title ("Vertical Alignment") -> I guess it is to make it clear for people because the label is with an abbreviation